From c8ad36290389e9369baeab2eba83eec7dfcf13c7 Mon Sep 17 00:00:00 2001
From: Paul Labedan <plabedan@gmail.com>
Date: Sun, 19 Jan 2014 14:44:10 +0100
Subject: [PATCH] fix install

---
 bgrive/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bgrive/CMakeLists.txt b/bgrive/CMakeLists.txt
index 074ab38..67850b3 100644
--- a/bgrive/CMakeLists.txt
+++ b/bgrive/CMakeLists.txt
@@ -40,4 +40,4 @@ set_target_properties( bgrive_executable
 )
 
 install(TARGETS bgrive_executable RUNTIME DESTINATION bin)
-install(FILES doc/grive.1 DESTINATION share/man/man1 )
+
-- 
1.8.5.2

